app/logic/userManagement.py

```diff
@@ -33,3 +33,27 @@ def changeCurrentTerm(term):
     newCurrentTerm.save()
 
     session["current_term"] = model_to_dict(newCurrentTerm)
+
+def addTerm():
```

Comment:
I would call this `addNextTerm`, so it is clearer how it works

Reply:
variable addTerm has been changed to addNextTerm